src/style.css

```diff
@@ -0,0 +1,113 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+  
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+html {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+}
+
+.search-bar-big {
+  font-weight: 300;
+  margin-top: 20px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  display: flex;
+  padding-top: 22px;
+  padding-bottom: 22px;
+     
+}
+
+.search-bar-small {
+  font-weight: 300;
+  margin-top: 20px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 42px;
+  padding-top: 7px;
+  padding-bottom: 12px;
+ 
+}
+
+.search-bar-small:focus,
+.search-bar-big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.icon_big {
+  width: 19px;
+  color: #3d4e61;
+  padding-left: 25px;
+  padding-right: 12.5px;
+  vertical-align: middle;
+        
+}
+
+.icon_small {
+  width: 11px;
+  color: #3d4e61;
+  padding-left: 12px;
+  vertical-align: middle;
+  padding-right: 4.5px;
+    
+}
+
+.input_big {
+  font-style: normal;
+  font-size: 16px;
+  line-height: 20px;
+  color: #3d4e61;
+  border: 1px solid #fff;
+  font-weight: 300;
+  font-family: inherit;
+  vertical-align: middle;
```

Comment:
you still duplicate many styles

Reply:
not fixed

Reply:
create common class and put there duplicated styles